app/boards/shields/eternal_keypad/eternal_keypad_lefty.keymap

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Unfortunately, I don't have a built version of the lefty, or an unbuilt pcb to test with.  Since this is specifically a gaming board, and the pcb is reversible, you can cheat by using the default transform and flipping A/D (to preserve WASD), and LEFT/RIGHT, then all the other keys will be in the correct locations upside down.

I did make a mistake here by forgetting to swap LEFT/RIGHT and adding the extra BT control keys to the ARROW layer, so I'll fix that before review.